### PR TITLE
A11y: Fix Fastpass issues for tracing query editors and config pages

### LINKED ISSUE #42714: Accessibility: Fix Jaeger query editor fastpass issues

### LINKED ISSUE #42715: Accessibility: Fix Jaeger config page fastpass issues

### LINKED ISSUE #42724: Accessibility: Fix Tempo query editor fastpass issues

### LINKED ISSUE #42725: Accessibility: Fix Tempo config page fastpass issues

### LINKED ISSUE #42726: Accessibility: Fix Zipkin query editor fastpass issues

### LINKED ISSUE #42727: Accessibility: Fix Zipkin config page fastpass issues

### PR DESCRIPTION
**Pull request by connorlindsey**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Relates to #41206
Fixes #42714
Fixes #42715 
Fixes #42724
Fixes #42725 
Fixes #42726
Fixes #42727

**Special notes for your reviewer**:
There are some global issues to fix:
- QueryField is not accessible and I couldn't find a great way to add a label
- RunButton needs a label when no text is supplied

### CODE REVIEW

File: public/app/core/components/NodeGraphSettings.tsx [hunk 1]
```diff
@@ -31,6 +31,7 @@ export function NodeGraphSettings({ options, onOptionsChange }: Props) {
           labelWidth={26}
         >
           <InlineSwitch
+            aria-label="Enable Node Graph"
```
Comment (kaydelaney) [inline public/app/core/components/NodeGraphSettings.tsx:34]: If we supplied an `id` prop here, we wouldn't need to repeat the label I think

File: public/app/core/components/TraceToLogsSettings.tsx [hunk 2]
```diff
@@ -124,6 +125,7 @@ export function TraceToLogsSettings({ options, onOptionsChange }: Props) {
           tooltip="Filters logs by Trace ID. Appends '|=<trace id>' to the query."
         >
           <InlineSwitch
+            aria-label="Filter by Trace ID"
```
Comment (kaydelaney) [inline public/app/core/components/TraceToLogsSettings.tsx:128]: Same as above

File: public/app/core/components/TraceToLogsSettings.tsx [hunk 3]
```diff
@@ -156,6 +159,7 @@ export function TraceToLogsSettings({ options, onOptionsChange }: Props) {
       <InlineFieldRow>
         <InlineField label="Loki Search" labelWidth={26} grow tooltip="Use this logs data source to search for traces.">
           <InlineSwitch
+            aria-label="Loki Search"
```
Comment (kaydelaney) [inline public/app/core/components/TraceToLogsSettings.tsx:162]: Same as above

File: public/app/plugins/datasource/jaeger/components/AdvancedOptions.tsx [hunk 4]
```diff
@@ -35,8 +35,10 @@ export function AdvancedOptions({ query, onChange }: Props) {
       >
         <div>
           <InlineFieldRow>
-            <InlineField label="Min Duration" labelWidth={21} grow>
+            <InlineField htmlFor="minDuration" label="Min Duration" labelWidth={21} grow>
               <Input
+                id="minDuration"
+                name="minDuration"
```
Comment (kaydelaney) [inline public/app/plugins/datasource/jaeger/components/AdvancedOptions.tsx:41]: Just supplying an `id` for `Input` should be enough I think, as `InlineField` looks at its children's props.

File: public/app/plugins/datasource/jaeger/components/AdvancedOptions.tsx [hunk 5]
```diff
@@ -49,8 +51,10 @@ export function AdvancedOptions({ query, onChange }: Props) {
             </InlineField>
           </InlineFieldRow>
           <InlineFieldRow>
-            <InlineField label="Max Duration" labelWidth={21} grow>
+            <InlineField htmlFor="maxDuration" label="Max Duration" labelWidth={21} grow>
               <Input
+                id="maxDuration"
```
Comment (kaydelaney) [inline public/app/plugins/datasource/jaeger/components/AdvancedOptions.tsx:56]: Same as above

File: public/app/plugins/datasource/jaeger/components/AdvancedOptions.tsx [hunk 6]
```diff
@@ -63,8 +67,16 @@ export function AdvancedOptions({ query, onChange }: Props) {
...
-            <InlineField label="Limit" labelWidth={21} grow tooltip="Maximum numbers of returned results">
+            <InlineField
+              htmlFor="limit"
+              label="Limit"
+              labelWidth={21}
+              grow
+              tooltip="Maximum numbers of returned results"
+            >
               <Input
+                id="limit"
```
Comment (kaydelaney) [inline public/app/plugins/datasource/jaeger/components/AdvancedOptions.tsx:78]: Same as above

File: public/app/plugins/datasource/jaeger/components/SearchForm.tsx [hunk 7]
```diff
@@ -54,6 +54,7 @@ export function SearchForm({ datasource, query, onChange }: Props) {
       <InlineFieldRow>
         <InlineField label="Service" labelWidth={14} grow>
           <Select
+            aria-label="Service"
```
Comment (kaydelaney) [inline public/app/plugins/datasource/jaeger/components/SearchForm.tsx:57]: `Select` accepts an `inputId` prop that `InlineField` checks so we don't have to repeat the label here

File: public/app/plugins/datasource/jaeger/components/SearchForm.tsx [hunk 8]
```diff
@@ -87,8 +89,10 @@ export function SearchForm({ datasource, query, onChange }: Props) {
         </InlineField>
       </InlineFieldRow>
       <InlineFieldRow>
-        <InlineField label="Tags" labelWidth={14} grow>
+        <InlineField htmlFor="tags" label="Tags" labelWidth={14} grow>
           <Input
+            id="tags"
+            name="tags"
```
Comment (kaydelaney) [inline public/app/plugins/datasource/jaeger/components/SearchForm.tsx:95]: Doesn't need `name`, I think?

File: public/app/plugins/datasource/jaeger/components/SearchForm.tsx [hunk 9]
```diff
@@ -72,6 +73,7 @@ export function SearchForm({ datasource, query, onChange }: Props) {
       <InlineFieldRow>
         <InlineField label="Operation" labelWidth={14} grow disabled={!query.service}>
           <Select
+            aria-label="Operation"
```
Comment (kaydelaney) [inline public/app/plugins/datasource/jaeger/components/SearchForm.tsx:76]: Same as above

File: public/app/plugins/datasource/tempo/QueryEditor/NativeSearch.tsx [hunk 10]
```diff
@@ -128,6 +128,7 @@ const NativeSearch = ({ datasource, query, onChange, onBlur, onRunQuery }: Props
         <InlineFieldRow>
           <InlineField label="Service Name" labelWidth={14} grow>
             <Select
+              aria-label="Service Name"
```
Comment (kaydelaney) [inline public/app/plugins/datasource/tempo/QueryEditor/NativeSearch.tsx:131]: Same as with the other `Select` components

File: public/app/plugins/datasource/tempo/QueryEditor/NativeSearch.tsx [hunk 11]
```diff
@@ -147,6 +148,7 @@ const NativeSearch = ({ datasource, query, onChange, onBlur, onRunQuery }: Props
         <InlineFieldRow>
           <InlineField label="Span Name" labelWidth={14} grow>
             <Select
+              aria-label="Span Name"
```
Comment (kaydelaney) [inline public/app/plugins/datasource/tempo/QueryEditor/NativeSearch.tsx:151]: Same as above

File: public/app/plugins/datasource/tempo/QueryEditor/NativeSearch.tsx [hunk 12]
```diff
@@ -237,6 +239,7 @@ const NativeSearch = ({ datasource, query, onChange, onBlur, onRunQuery }: Props
             tooltip="Maximum numbers of returned results"
           >
             <Input
+              aria-label="Limit"
```
Comment (kaydelaney) [inline public/app/plugins/datasource/tempo/QueryEditor/NativeSearch.tsx:242]: Just needs an `id` prop rather than `aria-label`

**Review by kaydelaney — COMMENTED** [review]
Thanks for this! Left some comments.

File: public/app/plugins/datasource/jaeger/components/AdvancedOptions.tsx (hunk 4 above)
Comment (connorlindsey) [reply public/app/plugins/datasource/jaeger/components/AdvancedOptions.tsx:41]: Gotcha, I didn't realize InlineField did that, that's pretty cool. I'll update these

**Review by kaydelaney — APPROVED** [review]

**Pull request merged**